test/test.js

```diff
@@ -37,4 +37,8 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {resolveImagePath: (value) => {return path.resolve(process.cwd(), value);}}, done);
+  });
```

Comment:
I’m totally sorry, I wasn’t installing with Yarn :( Everything works OK now.

As for the tests, I don’t how to formulate testing with spy with current test architecture. My approach seems kind of "icky", so I was wondering if you could take a look at it?